core_data_modules/data_models/messages_metrics.py

```diff
@@ -0,0 +1,47 @@
+from core_data_modules.data_models import validators
```

Comment:
Maybe this ought to live in metrics.py, so we could add other metrics here in future if we needed to.

Reply:
Sure, makes sense. I've renamed the file.